src/adapters/mqtt/index.ts

```diff
@@ -137,6 +138,19 @@ class MqttAdapter extends Adapter {
       const binding = operation.binding('mqtt')
       this.client.subscribe(channel, {
         qos: binding?.qos ? binding.qos : 0,
+      }, (err, granted) => {
+        if (err) {
+          logLineWithIcon('x', `Error while trying to subscribe to \`${channel}\` topic.`, {
+            highlightedWords: [channel],
+            iconColor: '#f00',
+            disableEmojis: true,
+          })
+          console.log(err.message)
+          return
+        }
+        logLineWithIcon(':zap:', `Subscribed to \`${channel}\` topic.`, {
```

Comment:
can you also use that granted object to show the QoS that has been granted?
the message should look something like:
`Subscribed to {topic} topic with QoS {QoS level}.`

Reply:
That should be doable 🙂

Reply:
Done :slightly_smiling_face: